tests/buyProduct_test.js

```diff
@@ -0,0 +1,38 @@
+const cart = require("../pages/cart");
+const { getProductPrice } = require("../pages/product");
+
+const USER = {
+    email: "test@test.qa",
+    password: "test",
+    firstName: "Name",
+    lastName: "Lastname",
+    address: "Address",
+    city: "City",
+    postcode: "666666",
+};
+
+
+
+Feature('purchase');
+
+Scenario.only('buy product', async ({ I, basePage, checkoutPage, productPage, cartPage }) => {
+    I.login(USER);
+    //await cartPage.cleanCart();
+    I.amOnPage("http://opencart.qatestlab.net/index.php?route=product/product&product_id=74");
+    productPage.chooseColor();
+    await productPage.getProductPrice();
```

Comment:
this one can be removed, duplicate of next row but without saving returned value